corehq/apps/reports/static/reports/js/bootstrap5/datatables_config.js

```diff
@@ -84,7 +85,7 @@ var HQReportDataTables = function (options) {
         });
         function applyBootstrapMagic() {
```

Comment:
This function name makes me queasy, but I admit it's pre-existing.

Reply:
eh...i agree but i'm just trying to do minimal changes to the original. i'm hoping we can just replace our reports altogether with an HTMX solution